<launch>

	 <arg name="joint_state" default="/arm5e/joint_state_angle" />
	 <arg name="joint_state_command" default="/arm5e/command_angle" />
	 <arg name="joint_state_fixed" default="/arm5e/joint_state_angle_fixed" />
	 		 
	 <param name="joint_state" value="$(arg joint_state)" />
	 <param name="joint_state_command" value="$(arg joint_state_command)" />
	 <param name="joint_state_fixed" value="$(arg joint_state_fixed)" />

     <node name="joint_offset_test" pkg="pm_manipulation" type="joint_offset_node" respawn="false" output="screen" />

</launch>
